android/build.gradle

```
@@ -147,6 +147,8 @@ android {
     // println "Native libs debug enabled: ${debugNativeLibraries}"
     doNotStrip debugNativeLibraries ? "**/**/*.so" : ''
     excludes = [
+      "META-INF",
+      "META-INF/**",
```

```suggestion
    excludes += [
```

the root cause is that we override the default exclusion list and `META-INF/MANIFEST.MF` is in the default list in fact. it's better for appending than overriding.